modules/git/tag.go

```diff
@@ -64,7 +64,7 @@ l:
 			}
 			nextline += eol + 1
 		case eol == 0:
-			tag.Message = string(data[nextline+1 : len(data)-1])
+			tag.Message = string(data[nextline : len(data)-1])
```

Comment:
Then the first char is '\n'?

Reply:
oh right .. I have to rm the other one :)